docs/api-reference/index.md

```diff
@@ -17,6 +17,9 @@
 .. autosummary::
    :toctree: ../generated/functions
    :recursive:
+
+   providers
+   amor.providers
```

Comment:
```suggestion
```

Not a top level function? Will be documented automatically via the submodule?

Reply:
Aha okay!